scripts/enum/items.lua

```diff
@@ -2054,18 +2065,33 @@ xi.items =
     BUNCH_OF_WILD_PAMAMAS           = 4596,
     BLACKENED_TOAD                  = 4599,
     LUCKY_EGG                       = 4600,
-    SCROLL_OF_DIA                   = 4606,
-    SCROLL_OF_STONE                 = 4607,
-    SCROLL_OF_CURE                  = 4608,
+    SCROLL_OF_DIA_EX                = 4606,
```

Comment:
Can you also please update the names in item_basic as well? That way tooling doesn't trip up in the future (just need the long name, not the sortname)

Reply:
Done